code/game/objects/effects/spawners/traps.dm

```diff
@@ -0,0 +1,10 @@
+/obj/effect/spawner/trap
+	name = "random trap"
+	icon = 'icons/obj/hand_of_god_structures.dmi'
+	icon_state = "trap_rand"
+
+/obj/effect/spawner/trap/Initialize()
```

Comment:
pretty sure this needs to be (mapload), as they currently aren't placing properly and leaving the 'random trap' effect.